docs/modules/ROOT/pages/erc20.adoc

```diff
@@ -17,17 +17,14 @@ See the {custom-decimals} guide.
 :dual-interfaces: xref:/interfaces.adoc#dual_interfaces[Dual interfaces]
 :erc20-supply: xref:/guides/erc20-supply.adoc[Creating ERC20 Supply]
 
-The following interface represents the full ABI of the Contracts for Cairo `ERC20` preset.
+The following interface represents the full ABI of the Contracts for Cairo `ERC20` component.
 The interface includes the `IERC20` standard interface as well as non-standard functions like `increase_allowance`.
-To support older deployments of ERC20, as mentioned in {dual-interfaces}, `ERC20` also includes the previously defined functions written in camelCase.
+To support older ERC20 token deployments, as mentioned in {dual-interfaces}, the `ERC20` component also includes the previously defined functions written in camelCase.
```

Comment:
```suggestion
To support older ERC20 token deployments, as mentioned in {dual-interfaces}, the `ERC20` component also includes an implementation of the interface written in camelCase.
```
the current version suggests that the functions are there by default (as it was with the preset)